src/Portalum.Zvt/ReceiveHandler.cs

```diff
@@ -106,11 +106,11 @@ public class ReceiveHandler : IReceiveHandler
         }
 
         /// <inheritdoc />
-        public ProcessDataState ProcessData(Span<byte> data)
+        public ProcessData ProcessData(Span<byte> data)
```

Comment:
in order for the `ProcessData` handler of the `ZVTCommunication` to cleanly differentiate between StatusInformation and other responses (e.g. ReceiptPrintout, ...) i am now returning the parsed message in addition to the parse state.